src/main/java/net/snowflake/client/core/HttpClientSettingsKey.java

```diff
@@ -150,4 +150,19 @@ public HttpProtocol getProxyHttpProtocol() {
   public Boolean getGzipDisabled() {
     return gzipDisabled;
   }
+
+  @Override
+  public String toString() {
+    return "HttpClientSettingsKey[" +
+           "ocspMode=" + ocspMode +
+           ", useProxy=" + useProxy +
+           ", proxyHost='" + proxyHost + '\'' +
+           ", proxyPort=" + proxyPort +
+           ", nonProxyHosts='" + nonProxyHosts + '\'' +
+           ", proxyUser='" + proxyUser + '\'' +
```

Comment:
should we add information that proxy password is set (without printing its content)?

Reply:
Good idea, added it.